utils/storybook/ComponentStatesTable.tsx

```diff
@@ -40,7 +40,7 @@ export interface ComponentStatesTableProps {
   children(props: Props): React.ReactNode;
 }
 
-const Table = styled('table')({
+export const Table = styled('table')({
```

Comment:
I didn't need all the states, as Badge isn't interactive, but this styled table was handy, so I exported it.